From 6b108e6dd62359809f627609fa091feb732f461c Mon Sep 17 00:00:00 2001
From: Chao Zeng <chao.zeng@siemens.com>
Date: Mon, 28 Dec 2020 15:32:49 +0800
Subject: [PATCH 25/28] iot2050: config the default device tree

modify the default device tree according to the new dts naming

Signed-off-by: Chao Zeng <chao.zeng@siemens.com>
---
 configs/iot2050_defconfig | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/configs/iot2050_defconfig b/configs/iot2050_defconfig
index 31931e7a5a..f2f2496c11 100644
--- a/configs/iot2050_defconfig
+++ b/configs/iot2050_defconfig
@@ -55,10 +55,10 @@ CONFIG_CMD_TIME=y
 # CONFIG_ISO_PARTITION is not set
 CONFIG_OF_CONTROL=y
 CONFIG_SPL_OF_CONTROL=y
-CONFIG_DEFAULT_DEVICE_TREE="iot2050-basic"
-CONFIG_OF_LIST="iot2050-basic iot2050-advanced"
+CONFIG_DEFAULT_DEVICE_TREE="k3-am6528-iot2050-basic"
+CONFIG_OF_LIST="k3-am6528-iot2050-basic k3-am6548-iot2050-advanced"
 CONFIG_SPL_MULTI_DTB_FIT=y
-CONFIG_SPL_OF_LIST="iot2050-spl"
+CONFIG_SPL_OF_LIST="k3-am65-iot2050-spl"
 CONFIG_SPL_MULTI_DTB_FIT_NO_COMPRESSION=y
 CONFIG_ENV_IS_IN_SPI_FLASH=y
 CONFIG_SYS_REDUNDAND_ENVIRONMENT=y
-- 
2.25.1

